src/sentry/releasehealth/sessions.py

```diff
@@ -50,6 +52,17 @@ def get_release_adoption(
             project_releases=project_releases, environments=environments, now=now
         )
 
+    def get_release_sessions_time_bounds(
+        self,
+        project_id: ProjectId,
+        release: ReleaseName,
+        org_id: OrganizationId,
+        environments: Optional[Sequence[EnvironmentName]] = None,
+    ) -> ReleaseSessionsTimeBounds:
+        return _get_release_sessions_time_bounds(  # type: ignore
```

Comment:
I don't understand why you need a `type: ignore` here? Doesn't `_get_release_sessions_time_bounds` return `ReleaseSessionsTimeBounds`?

Reply:
Yeah but I really don't want to get into the business of refactoring adjacent modules

Reply:
but its not adjacent modules.. you are calling that refactoring this very specific module 
Surely if its something big I understand but if its a low hanging fruit like adding something like
`def get_release_sessions_time_bounds(project_id: int, release: str, org_id: int, environments: Optional[List[str]]=None) -> _TimeBounds :`

shouldn't be that big of a deal

Reply:
You can't just put the type signature on there, you need to convert the entire file and add it to mypy.ini for mypy to even consider it for typing.